File: app/src/main/java/com/lkms/data/model/java/Equipment.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Đại diện cho một hàng trong bảng "Equipment".
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Equipment {

    @SerializedName("equipmentId")
    private Integer equipmentId;

    @SerializedName("equipmentName")
    private String equipmentName;

    @SerializedName("model")
    private String model;

    @SerializedName("serialNumber")
    private String serialNumber;

    @SerializedName("availability")
    private Boolean availability;
}

File: app/src/main/java/com/lkms/data/model/Equipment.kt
// Trong file: data/model/Equipment.kt
package com.lkms.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Đại diện cho một hàng trong bảng "Equipment".
 */
@Serializable
data class Equipment(
    @SerialName("equipmentId")
    val equipmentId: Int? = null,

    @SerialName("equipmentName")
    val equipmentName: String? = null,

    @SerialName("model")
    val model: String? = null,

    @SerialName("serialNumber")
    val serialNumber: String? = null,

    @SerialName("availability")
    val availability: Boolean? = null
)